### PR TITLE
Delete series confirmation to be different to delete episode

### PR DESCRIPTION
**Pull request by nizans**
**Changes**

Update the delete text in item context menu and confirmation modal when the item is a series to emphasize deletion of the entire series with all its episodes.

**Issues**

Should close https://features.jellyfin.org/posts/2573/delete-series-confirmation-to-be-different-to-delete-episode

### CODE REVIEW

File: src/components/itemContextMenu.js
Comment (hdavidkim) [inline src/components/itemContextMenu.js:14]: nit: i believe "label" is usually the terminology used for webapp text elements

```suggestion
function getDeleteLabel(type) {
```

File: src/components/itemContextMenu.js
Comment (hdavidkim) [inline src/components/itemContextMenu.js:18]: mind adding `'Episode'` as well given that it was requested in the original thread?

File: src/components/itemContextMenu.js
Comment (hdavidkim) [inline src/components/itemContextMenu.js:15]: in a file littered with if/else's, i thank you for using a switch lol

File: src/scripts/deleteHelper.js
Comment (hdavidkim) [inline src/scripts/deleteHelper.js:12]: renaming to something like `getConfirmContent`/`getDeletionConfirmContent` and returning `title`, `text`, `confirmText`, as well as `primary` would go a long way into reducing excess code within `deleteItem` as you can just call this function directly below e.g. `return confirm(getConfirmContent(item))`

**Review by hdavidkim — COMMENTED** [review]
thank you for working on this PR and lmk if you have any questions about my comments

**Review by nielsvanvelzen — COMMENTED** [review]
Translation files should only be changed via our translation tool (translate.jellyfin.org) to avoid conflicts. Please remove the changes to the hebrew translation, you can re-add those via the translation tool once the PR is merged.

File: src/scripts/deleteHelper.js
Comment (nizans) [reply src/scripts/deleteHelper.js:12]: Not sure if it is better especially when there it is not a TS file, but I dont mind applying your suggestion. Done.

File: src/components/itemContextMenu.js
Comment (nizans) [reply src/components/itemContextMenu.js:14]: Sure, Updated

File: src/components/itemContextMenu.js
Comment (nizans) [reply src/components/itemContextMenu.js:15]: :D

File: src/components/itemContextMenu.js
Comment (nizans) [reply src/components/itemContextMenu.js:18]: Sure thing, Done!.

**Comment by nizans** [comment]
> thank you for working on this PR and lmk if you have any questions about my comments

Thanks you @hdavidkim for reviewing this PR. I added a comment with your suggestions. Please take a second look if you can.

**Comment by nizans** [comment]
> Translation files should only be changed via our translation tool (translate.jellyfin.org) to avoid conflicts. Please remove the changes to the hebrew translation, you can re-add those via the translation tool once the PR is merged.

@nielsvanvelzen NP, removed `he` translations from this PR.

**Review by hdavidkim — APPROVED** [review]
ty for the follow up. this looks good on my end!

**Comment by nizans** [comment]
@nielsvanvelzen Hey,
How can we get this approved by a maintainer?

**Comment by thornbill** [comment]
Thanks for the review @hdavidkim! I will try to look over this today.

File: src/components/itemContextMenu.js
Comment (thornbill) [inline src/components/itemContextMenu.js:16]: Please use `BaseItemKind` from the SDK for these values.

```js
import { BaseItemKind } from '@jellyfin/sdk/lib/generated-client/models/base-item-kind';
```

Comment (nizans) [reply src/components/itemContextMenu.js]: Nice, I didn't know it existed.
Done

**Review by thornbill — APPROVED** [review]
I think there is an opportunity to also treat deleting seasons and albums in a similar fashion to series, but this is a great start!

**Comment by nizans** [comment]
> I think there is an opportunity to also treat deleting seasons and albums in a similar fashion to series, but this is a great start!

Sounds good, I'll open a separate PR

**Comment by nizans** [comment]
@thornbill Thanks for the merge and approval

**Comment by CabbageSmell** [comment]
On my google tv, I just found this thread because I deleted a series for the 3rd time by accident.  I just want to delete the episode when I am finished watching but after I complete watching an episode I am taken automatically to the Series landing page and the delete button does not say what it is deleting. Lol, this is a bit painful.  Can you please dummyproof the delete button for me?

**Pull request merged**
